server/handlers/default.go

```diff
@@ -108,7 +110,19 @@ func atomicUpdateHandler(ctx context.Context, w http.ResponseWriter, r *http.Req
 		logger.Errorf("500 POST error applying update request: %v", err)
 		return errors.ErrUpdating.WithDetail(nil)
 	}
-	return nil
+
+	broadcasterKey := ctx.Value("broadcaster")
+	broadcaster, ok := broadcasterKey.(*dNotifications.Broadcaster)
+	if !ok {
+		return errors.ErrUnknown.WithDetail("could not get notification broadcaster")
+	}
+	sourceRecordKey := ctx.Value("sourceRecord")
+	sourceRecord, ok := sourceRecordKey.(dNotifications.SourceRecord)
+	if !ok {
+		return errors.ErrUnknown.WithDetail("could not get notification sourceRecord")
+	}
+
+	return notifications.Notify("GUN update", sourceRecord, "Josh", r, broadcaster)
```

Comment:
Need to update the username that we pass to Notify, might try to parse it out of the JWT claims